tensorboard/plugins/core/core_plugin.py

```diff
@@ -285,6 +292,31 @@ def _serve_experiment_runs(self, request):
         results = []
         return http_util.Respond(request, results, "application/json")
 
+    @wrappers.Request.application
+    def _serve_notifications(self, request):
```

Comment:
Let's add at least all minimal test for this in `core_plugin_test.py` - you could model it off this one:

https://github.com/tensorflow/tensorboard/blob/9bb887b334902e6b638ea3620d8203e05b700b78/tensorboard/plugins/core/core_plugin_test.py#L298-L302

(The fact that `experiment_runs` has no test is a bit anomalous; its test was removed a while ago and the endpoint should probably also have been removed then, but was left in place.)

Reply:
add one arbitrary test. let me know if that is enough